### PR TITLE
Sso login

### PR DESCRIPTION
**Pull request by kdrtibor**

### CODE REVIEW

File: OctaneVSPlugin/ConnectionSettings.cs [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+				// create a new service object
+				OctaneServices.Create(OctaneConfiguration.Url,
+						  OctaneConfiguration.SharedSpaceId,
+						  OctaneConfiguration.WorkSpaceId);
+
+                
+                // close all opened details windows so that we don't have details windows
+                // for entities from different workspaces
+                PluginWindowManager.CloseAllDetailsWindows();
```
Comment (atothhpe) [inline OctaneVSPlugin/ConnectionSettings.cs]: need to find a way to fix the messed up indentation in vs

File: OctaneVSPlugin/ConnectionSettings.cs [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+		}
+
+        public async void InitialisePluginComponents()
+        {
+            await OctaneServices.GetInstance().Connect();
+            await OctaneMyItemsViewModel.Instance.LoadMyItemsAsync();
+            await EntityTypeRegistry.Init();
+        } 
+
+		public async Task<string> TestConnection()
```
Comment (atothhpe) [inline OctaneVSPlugin/ConnectionSettings.cs]: indentation driving me crazy

File: OctaneVSPlugin/MainWindowPackage.cs [hunk 3]
```diff
@@ -48,7 +48,8 @@ namespace octane_visual_studio_plugin
     [ProvideToolWindow(typeof(DetailsToolWindow), MultiInstances = true, Style = VsDockStyle.Tabbed, Window = MainWindow.WINDOW_ID, Transient = true)]
     [ProvideToolWindow(typeof(SearchToolWindow), MultiInstances = true, Style = VsDockStyle.Tabbed, Window = MainWindow.WINDOW_ID, Transient = true)]
     [Guid(MainWindowPackage.PackageGuidString)]
-    [ProvideOptionPage(typeof(OptionsPage), "ALM Octane", "Server Details", 0, 0, true)]
+    //[ProvideOptionPage(typeof(OptionsPage), "ALM Octane", "Server Details", 0, 0, true)]
```
Comment (atothhpe) [inline OctaneVSPlugin/MainWindowPackage.cs]: comment obsolete?

File: OctaneVSPlugin/ConnectionSettings.cs (hunk 1 above)
Comment (kdrtibor) [reply OctaneVSPlugin/ConnectionSettings.cs]: github is messing with the indentation
looks ok to me
image

File: OctaneVSPlugin/ConnectionSettings.cs (hunk 2 above)
Comment (kdrtibor) [reply OctaneVSPlugin/ConnectionSettings.cs]: the file is messed up for some reason by github

**Pull request merged**